src/internal/base-component/metrics/log-clients.ts

```diff
@@ -91,28 +97,26 @@ export class PanoramaClient {
     }
     if (!validateLength(metric.eventName, 1000)) {
       console.error(`Event name for metric is too long: ${metric.eventName}`);
-      return;
+      return true;
```

Comment:
Validation constraints are same for both API, return `true` to prevent it from attempting to use AWSC.Clog.log if the validation fails